.circleci/config.yml

```diff
@@ -545,12 +555,13 @@ jobs:
       - checkout
       - designate_upload_channel
       - load_conda_channel_flags
+      - attach_workspace:
+          at: .
       - run:
           name: Generate cache key
           # This will refresh cache on Sundays, nightly build should generate new cache.
           command: |
             echo "$(date +"Year-Week %Y-%U")" > .circleci-weekly
```

Comment:
Shouldn't this command then also suffer from the same issue? Do we even need the weekly cache anymore?

Reply:
Not really, because the key unique owned by the job, and does not rely on any other cache building-up job (like the case for daily cache). 

will do a follow-up on removing the weekly cache by consolidating everything into daily cache.